README.md

```diff
@@ -105,6 +92,15 @@ Run `juju switch <juju model>` to switch to the target Juju model.
 juju status --relations
 ```
 
+#### Identity Platform Admin UI, OpenFGA, External IdP Integrator
+
+If the extra charms were not deployed as part of the platform in the previous step, they can be added later with:
+```
+terraform -chdir=modules/extra init
+terraform -chdir=modules/extra apply -var-file="../../vars.tfvars" -var deploy_extra_charms=true
```

Comment:
We shouldn't have to apply the same vars file to the `iam` and the `extra` modules.